js/Update.elm

```diff
@@ -8,13 +8,15 @@ import AnimationFrame
 import Time exposing (Time)
 import Debug
 import Server
+import Shake
 
 
 subscriptions : Model -> Sub Msg
 subscriptions model =
     Sub.batch
         [ Server.listen model ServerMsgReceived
         , AnimationFrame.times AnimationFrame
+        , Shake.shake (always Shake)
```

Comment:
Better put this inside the TradeStage case

Reply:
Done